### PR TITLE
Progress bar color through prop

### PR DESCRIPTION
**Pull request by VonLatvala**
Can be set by specifying `barColor` prop in `ProgressBar` component

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -260,7 +260,7 @@ class App extends React.Component {
 
         return (
             <div style={style.body}>
-                <ProgressBar />
+                <ProgressBar barColor={'#2696fb'} />
```
Comment (VonLatvala) [inline src/components/App.jsx]: don't fsckin hardcode

**Review by andersnylund — CHANGES_REQUESTED** [review]
On Chrome this works fine until the progress of the app reaches the The thank you page. There the progress bar is set back to the original darker blue color. 

And on Firefox the functionality breaks the app completely
Screenshot 2019-04-08 at 8 09 32

**Comment by VonLatvala** [comment]
> On Chrome this works fine until the progress of the app reaches the The thank you page. There the progress bar is set back to the original darker blue color.
> 
> And on Firefox the functionality breaks the app completely
> Screenshot 2019-04-08 at 8 09 32

Firefox was never a target platform, but yeah..

**Review by andersnylund — APPROVED** [review]
Could the same be done with the circular progress bar?

**Comment by VonLatvala** [comment]
> Could the same be done with the circular progress bar?

nope, not supported, support will be added in a later release

**Comment by VonLatvala** [comment]
> Could the same be done with the circular progress bar?

Oh, sorry, yes, most probably, the same hack could be done to the circular one

**Pull request merged**